components/gcp/bigquery/query/bigquery_query_to_gcs_op.yaml

```diff
@@ -0,0 +1,83 @@
+# Export to bucket in gcs
```

Comment:
Can you please put each `component.yaml` into its own sub-directory. For example `components/gcp/bigquery/query/to_GCS/component.yaml`. This way the components are easier to search and discover (by filename) and each one can have it's own readme.